### DISCUSSION TITLE
Hide full buffer path from buffer picker

### DISCUSSION THREAD

**Discussion by jfaz13** [helix-editor/helix#10204] [General]
I find having file paths for already-opened buffers is pretty redundant and makes it tedious to read through the buffer list in order to find an already-opened buffer to go back to.
Instead of:
`src\app\test\test.x`
`src\app\test\test2.x`
`src\app\test\test3.x`
`src\app\test\test4.x`
`src\app\test2\test.x`

Simply having:
`src\app\test\test.x`
`test2.x`
`test3.x`
`test4.x`
`src\app\test2\test.x`

Like most IDEs and text editors do with their tabs. Only show full paths when paths differ and they have the same name. Maybe diff'd paths could be shown to shorten it further, e.g:
`test\test.x`
`test2.x`
`test3.x`
`test4.x`
`test2\test.x`

**Comment by the-mikedavis** [comment]
The full path is listed for searching. It sounds like https://github.com/helix-editor/helix/pull/2377 would cover what you're looking for

**Comment by jfaz13** [comment]
> The full path is listed for searching. It sounds like #2377 would cover what you're looking for

Definitely looks great, however that file explorer would only show files under a directory, when in my case I have files all over the place that I'm switching between and having the full path is distracting (especially when it gets long). It's really the only thing that feels a bit jarring switching from a standard tab-based editor like VSCode which only displays the filename. It just makes finding the file I was editing a few minutes ago a bit faster. Do people actually use the buffer picker to search for full paths? I would imagine most use the fuzzy finder for that, as if you can't remember the actual filename you might not even have a buffer open for it anyways. And if the files conflict then you get the path anyways.
